.gitignore

```diff
@@ -1,2 +1,4 @@
 .DS_Store
 node_modules
+crossfilter.js
+crossfilter.min.js
```

Comment:
Really nitpicking here, but files should end with an empty line.

Reply:
If nothing else, it would cause a spurious change later when opened in many editors.

Reply:
ab083ed